docs/reference-docs/forms.md

```diff
@@ -219,7 +225,7 @@ export function autoFieldFunction(props: GuaranteedProps<unknown> & Record<strin
 
 ```
 
-### custom field example
+#### Custom Field Example
 
 example custom field to select a user by group.
 
```

Comment:
I think that the idea was that we'd **_generally_** do the v2 docs entirely separately (I think Wouter is finding someone to do that on the SURF side) since it's a separate library. The cool thing is, if we do that, we can still use mkdocstrings if we turn on the inventory for that repo:

https://mkdocstrings.github.io/usage/#cross-references-to-other-projects-inventories


(also assumes there is a js handler, which I'm not sure on, but boy, that'd be fun to build!)